app/vmagent/remotewrite/remotewrite.go

```diff
@@ -216,12 +218,19 @@ func Init() {
 	// Start config reloader.
 	configReloaderWG.Add(1)
 	go func() {
+		var tickerCh <-chan time.Time
+		if *remoteWriteConfigCheckInterval > 0 {
+			ticker := time.NewTicker(*remoteWriteConfigCheckInterval)
+			tickerCh = ticker.C
+			defer ticker.Stop()
+		}
 		defer configReloaderWG.Done()
 		for {
 			select {
 			case <-configReloaderStopCh:
 				return
 			case <-sighupCh:
+			case <-tickerCh:
```

Comment:
this would print `reloading relabel configs pointed by -remoteWrite.relabelConfig and -remoteWrite.urlRelabelConfig` message every remoteWrite.configCheckInterval.
Please refer to how `promscrape.configCheckInterval` is done - it doesn't generate extra logs and doesn't increment metrics unless something has changed.

cc @makasim